components/EmotionRecordsList.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const StyledList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0 auto 1rem;
+  text-align: left;
+`;
+
+const StyledListItem = styled.li`
+  margin: 1rem auto;
+  border: 1px solid black;
+  border-radius: 5px;
+  padding: 1rem;
+  box-shadow: 0 0 3px 0;
+  cursor: pointer;
+
+  &:hover {
+    background-color: lightskyblue;
+  }
+`;
+
+const StyledDetails = styled.div`
+  display: ${({ $show }) => ($show ? "block" : "none")};
+  padding: 0 1rem;
+  margin-bottom: 2rem;
+`;
```

Comment:
⚠️ `StyledDetails` is currently styled as a `<div>` and might be used directly within a `<ul>`, which is not compliant with HTML standards. 
Consider changing `StyledDetails` to be styled as an `<li>` or wrap it in an `<li>` element to ensure valid HTML structure.